rust/tvm-rt/src/function.rs

```diff
@@ -113,8 +114,11 @@ impl Function {
     /// Calls the function that created from `Builder`.
     pub fn invoke<'a>(&self, arg_buf: Vec<ArgValue<'a>>) -> Result<RetValue> {
         let num_args = arg_buf.len();
-        let (mut values, mut type_codes): (Vec<ffi::TVMValue>, Vec<ffi::TVMArgTypeCode>) =
-            arg_buf.into_iter().map(|arg| arg.to_tvm_value()).unzip();
+        let (mut values, mut type_codes): (Vec<ffi::TVMValue>, Vec<ffi::TVMArgTypeCode>) = arg_buf
+            .clone()
```

Comment:
I don't see this value being reused later?

Reply:
I think this is was when I was doing some debugging on this.

Reply:
Removed